### ISSUE TITLE
tensorflow and theano show different error about Concatenate function using 2.0

### ISSUE THREAD

**Issue by hachangho** [fchollet/keras#7377]
about this source 

it works fine on theano. 
---------------------------------
K.set_image_dim_ordering('th')
inputs = Input((n_ch, patch_height, patch_width))

but tensorflow show below error
---------------------------------
K.set_image_dim_ordering('tf')
inputs = Input((patch_height, patch_width, n_ch))
<Error>
`Concatenate` layer requires inputs with matching shapes except for the concat axis. Got inputs shapes: [(None, 24, 24, 128), (None, 24, 24, 64)]

< Source >
K.set_image_dim_ordering('tf')
inputs = Input((patch_height, patch_width, n_ch))

conv1 = Conv2D(32, (3, 3), activation="relu", padding="same")(inputs)
conv1 = Dropout(0.2)(conv1)
conv1 = Conv2D(32, (3, 3), activation="relu", padding="same")(conv1)
pool1 = MaxPooling2D(pool_size=(2, 2))(conv1)

conv2 = Conv2D(64, (3, 3), activation="relu", padding="same")(pool1)
conv2 = Dropout(0.2)(conv2)
conv2 = Conv2D(64, (3, 3), activation="relu", padding="same")(conv2)
pool2 = MaxPooling2D(pool_size=(2, 2))(conv2)

conv3 = Conv2D(128, (3, 3), activation="relu", padding="same")(pool2)
conv3 = Dropout(0.2)(conv3)
conv3 = Conv2D(128, (3, 3), activation="relu", padding="same")(conv3)
upsam1 = UpSampling2D(size=(2, 2))(conv3) 
**up1 = Concatenate(axis=1)([upsam1, conv2])  # <-- this line make error**

**Comment by Dref360** [comment]
axis should be -1 for a `channels_last` ordering I think.

**Comment by hachangho** [comment]
thanks Dref360. it works fine.
